test/extended/images/mongodb_ephemeral.go

```diff
@@ -39,22 +45,10 @@ var _ = g.Describe("[images][mongodb] openshift mongodb image", func() {
 			o.Expect(podNames).Should(o.HaveLen(1))
 
 			g.By("expecting the mongodb service is answering for ping")
-			mongo := db.NewMongoDB(podNames[0], "")
-
-			for times := 0; times < 10; times++ {
-				ok, err := mongo.IsReady(oc)
-				if ok {
-					break
-				}
-
-				if times == 10 {
-					o.Expect(err).ShouldNot(o.HaveOccurred())
-					o.Expect(ok).Should(o.BeTrue())
-					break
-				}
-
-				time.Sleep(1 * time.Second)
-			}
+			mongo := db.NewMongoDB(podNames[0])
+			ok, err := mongo.IsReady(oc)
```

Comment:
> what it should do is to check whether the mongo service has endpoint(s). if it does, that means the readiness check passed and you are good to send queries.

We had a checking for endpoints by in current implementation I need to also wait when mongo will be ready. Until readiness check will be merged I can't use it. I can put the comment with TODO entry here. Are you agree?